### PR TITLE
Add consumer given values to logs

### PR DESCRIPTION
**Pull request by enxebre**
Add consumer given values to logs

### CODE REVIEW

File: csr_check.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ var nodeBootstrapperGroups = sets.NewString(
 
 func validateCSRContents(req *certificatesv1beta1.CertificateSigningRequest, csr *x509.CertificateRequest) (string, error) {
 	if !strings.HasPrefix(req.Spec.Username, nodeUserPrefix) {
-		return "", fmt.Errorf("Doesn't match expected prefix")
+		return "", fmt.Errorf("%q doesn't match expected prefix: %q", req.Spec.Username, nodeUserPrefix)
```
Comment (ingvagabund) [inline csr_check.go]: @cgwalters did you omit the values on purposes due to limiting the data that gets logged due to security reasons? Or, in the time you added better logging message [1] it was not necessary?

[1] https://github.com/openshift/cluster-machine-approver/commit/374b20416ec80f8a05a5b178d6b2313553c04a0f#diff-a482a1a664767249971b0b71ca9baaf0

Comment (cgwalters) [reply csr_check.go]: Nope, I didn't think about it too much.  I don't see a reason not to log more data, only kubeadmin should have access to the CSR logs right?

Comment (ingvagabund) [reply csr_check.go]: I assume that's true in production. Thanks.

**Pull request merged**